manage.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+"""Django's command-line utility for administrative tasks."""
+import os
+import sys
+
+
+def main():
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'simpact.settings')
```

Comment:
Can you please add instructions how to run Django server from the command line to README? This way people won't have to go through Django docs to figure this out.